lib/gooddata/bricks/provisioning_brick.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'brick'
+
+module GoodData::Bricks
```

Comment:
Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.